source/Mlos.Proto/OptimizerService.proto

```diff
@@ -67,6 +71,14 @@ message RegisterObservationRequest
     Observation Observation = 2;
 };
 
+// A request to add a observations to the optimizer's data set.
+//
+message RegisterObservationsRequest
+{
+    OptimizerHandle OptimizerHandle = 1;
+    Observations Observations = 2;

```

Comment:
I think we talked about this already at one point, but can you please remind me why we need the difference for a single Observation and and Observations instead of an array of Observation objects (e.g. using `repeated`)?

Reply:
In short: column vs. row layout. Observations end up being a dictionary of lists, rather than a list of dictionaries and as a result are faster to (de)serialize, occupy less space and are more easily ingested by pandas.

Reply:
Should probably add some comments to that effect.